tests/src/Interop/PInvoke/IEnumerator/IEnumeratorNative.cpp

```diff
@@ -0,0 +1,103 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "IEnumeratorNative.h"
+#include <xplatform.h>
+
+extern "C" DLL_EXPORT HRESULT STDMETHODCALLTYPE GetIntegerEnumerator(int start, int count, IEnumVARIANT** ppEnum)
+{
+    if (count < 0)
+    {
+        return E_INVALIDARG;
+    }
+
+    *ppEnum = new(CoreClrAlloc(sizeof(IntegerEnumerator))) IntegerEnumerator(start, count);
```

Comment:
Placement new is no longer valid and will more than likely crash now. Same below.